tests/integration_tests/data_steward/tools/load_vocab_test.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import os
+from pathlib import Path
+import mock
+
+from google.cloud import storage, bigquery
+
+import app_identity
+from tests.test_util import TEST_VOCABULARY_PATH
+from common import CONCEPT, VOCABULARY
+from tools import load_vocab as lv
+
+
+class LoadVocabTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = app_identity.get_application_id()
+        self.dataset_id = os.environ.get('UNIONED_DATASET_ID')
+        self.staging_dataset_id = f'{self.dataset_id}_staging'
+        self.bucket = os.environ.get('BUCKET_NAME_FAKE')
+        self.bq_client = bigquery.Client(project=self.project_id)
+        self.gcs_client = storage.Client(project=self.project_id)
+        self.test_vocab_folder_path = Path(TEST_VOCABULARY_PATH)
+        self.test_vocabs = [CONCEPT, VOCABULARY]
+        self.contents = {}
+        for vocab in self.test_vocabs:
+            vocab_path = self.test_vocab_folder_path / lv._table_name_to_filename(
+                vocab)
+            with vocab_path.open('r') as f:
+                self.contents[vocab] = f.read()
+
+    @mock.patch('tools.load_vocab.VOCABULARY_TABLES', [CONCEPT, VOCABULARY])
+    def test_upload_stage(self):
+        lv.main(self.project_id, self.bucket, self.test_vocab_folder_path,
+                self.dataset_id)
+        expected_row_count = {CONCEPT: 101, VOCABULARY: 52}
+        for dataset in [self.staging_dataset_id, self.dataset_id]:
+            for vocab in self.test_vocabs:
+                content_query = f'SELECT * FROM `{self.project_id}.{dataset}.{vocab}`'
+                content_job = self.bq_client.query(content_query)
+                rows = content_job.result()
+                self.assertEqual(len(list(rows)), expected_row_count[vocab])
+
```

Comment:
Do we want to include tests for some of the other methods in the module?

Reply:
I kept only one test for integration to test the full pipeline. I could add a few more unit tests.

Reply:
Up to you, was just wondering. I'll approve, and you can decide.